services/fishAges.service.ts

```diff
@@ -29,7 +30,12 @@ export type FishAge<
 
 @Service({
   name: 'fishAges',
-  mixins: [DbConnection()],
+  mixins: [DbConnection({
+    collection: 'fishAges',
```

Comment:
https://github.com/AplinkosMinisterija/biip-zuvinimas-api/blob/main/mixins/database.mixin.ts#L13 please update it here - `tableName: opts.collection` because knex don't have `collection` option but `tableName` exists.